tests/python/common/transforms/test_to_block.py

```diff
@@ -0,0 +1,192 @@
+##
```

Comment:
Remove the duplicated test in https://github.com/dmlc/dgl/blob/master/tests/python/common/transforms/test_transform.py

Reply:
removed.